api/index.js

```diff
@@ -13,15 +13,22 @@ app.use(express.json());
 app.use(free_the_cors());
 app.use(braidify);    // Add braid stuff to req and res
 
-const board = {};
-for (let i = 1; i < 11; i++) {
-  for (let j = 1; j < 11; j++) {
-    board[`${i}_${j}`] = ".";
+const boards = {}
+
+const generateBoard = (url) => {
+  boards[url] = {}
+  for (let i = 1; i < 11; i++) {
+    for (let j = 1; j < 11; j++) {
+      boards[url][`${i}_${j}`] = ".";
+    }
   }
 }
 
-app.get("/api/board", (req, res) => {
+
+app.get("/api/:url", (req, res) => {
   // res.json(board);
+  let url = req.params.url;
+  generateBoard(url)
```

Comment:
this will set the board as empty on every GET request, even if it already exists. don't do that.

instead check if `boards[url]` is already defined before generating a new board.